bpfman-operator/Makefile

```diff
@@ -109,6 +109,9 @@ ENVTEST ?= $(LOCALBIN)/setup-envtest
 CM_VERIFIER ?= $(LOCALBIN)/cm-verifier
 OPERATOR_SDK ?= $(LOCALBIN)/operator-sdk
 
+GOARCH ?= amd64
```

Comment:
Can you make this automatic based on the build system? 

i.e use the output of 
```
go env GOOS GOARCH
linux
amd64
```

Reply:
Done